runtimes/ruby/ruby-on-rails.md

```diff
@@ -43,17 +44,15 @@ Then, you need to provide a Gemfile.lock. To do that, please run:
 .../myapp $ git add Gemfile.lock
 .../myapp $ git commit -m "Add production dependency file"
 ```
+## Environment injection
 
-## Choose ruby version
-
-Simply add this to your Gemfile
+To access your variable in your application, nothing simpler! Just get
+it from your environment, like you would with `PATH`:
+`ENV["MY_VARIABLE"]`.
 
-```
-ruby '2.3'
-```
+### Choose ruby version
 
-`rubyversion` must be a string. The given string is tested against the
-available versions by using the /^<ver>/ regexp.
+On your Clever application create an `RUBY_VERSION=rubyversion` where `rubyversion` represents:
```

Comment:
I think it's best to use "Clever Cloud" here, maybe saying "in the console" or something like that.

Also, "create an environment variable `RUBY_VERSION=rubyversion`